docs/Getting-Started-TPU-VM.md

```diff
@@ -115,17 +115,19 @@ subnetwork: "default"
 EOF
 ```
 
+If you want to custom the docker image that is created and uploaded to GCP Artifact Registry, you can add config `image_name: "YOUR-DOCKER-NAME"`.
+
 Now run `launch.py`. This will package your current directory into a Docker image and run it on your workers. Everything after the `--` is run on each worker.
 
 ```bash
-python infra/launch.py -- python levanter/src/levanter/main/train_lm.py --config_path levanter/config/gpt2_small.yaml --trainer.checkpointer.base_path gs://<somewhere>'
+python infra/launch.py -- python src/levanter/main/train_lm.py --config_path config/gpt2_small.yaml --trainer.checkpointer.base_path gs://<somewhere>'
```

Comment:
The target is already set to be `/home/levanter`, so we don't need to include `levanter` in the beginning of the path.

Reply:
Thanks for catching!